new/LIT7052Qene.xml

```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7052Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragalā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
```

Comment:
Thank you! I would say: In the world, he is not on the chariot but on the back of the donkey, the speaker of the good things. ...

saragalla with ll

Reply:
I assume, that "kwellekkemu type" is [Kʷǝllǝkǝmu gǝʿǝz](https://betamasaheft.eu/authority-files/list?keyword=Kwellekomu) in the list of Qene-types on the Bm-website.